Sources/Extensions/CollectionViewManager+UICollectionViewDataSource.swift

```diff
@@ -26,10 +26,14 @@ extension CollectionViewManager: UICollectionViewDataSource {
     open func collectionView(_ collectionView: UICollectionView,
                              viewForSupplementaryElementOfKind kind: String,
                              at indexPath: IndexPath) -> UICollectionReusableView {
-        //TODO: Add same `configure` logic as with cell items
-        let reusableViewItem = sectionItem(for: indexPath)?.reusableViewItems.filter { $0.type.kind == kind }.first
-        let view = reusableViewItem?.view(for: collectionView, at: indexPath)
-        return view ?? UICollectionReusableView()
+        guard let viewItem = self.viewItem(for: indexPath) else {
+            return UICollectionReusableView()
+        }
+        let view = collectionView.dequeueReusableSupplementaryView(ofKind: viewItem.type.kind,
```

Comment:
I'm not sure if reusable views can have same `indexPath` but different `kind` BUT(!) I think it would be better to identify reusable view items not only by `indexPath` anyway 🤔